### ISSUE TITLE
ompl_multisets fail in a catkin build workspace

### ISSUE THREAD

**Issue by mkoval** [personalrobotics/multiset-planning#3]
```
    Start 4: _ctest_ompl_multiset_gtest_test_e8simple                                                                                                                                                       
4/5 Test #4: _ctest_ompl_multiset_gtest_test_e8simple ..............***Failed    0.11 sec                                                                                                                   
    Start 5: _ctest_ompl_multiset_gtest_test_e8fromfile                                                                                                                                                     
5/5 Test #5: _ctest_ompl_multiset_gtest_test_e8fromfile ............***Failed    0.11 sec                                                                                                                   
```

**Comment by mkoval** [comment]
Both of those tests pass, but then SEGFAULT if I run them manually:

```
$ rosrun ompl_multiset test_e8simple 
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from E8SimpleTestCase
[ RUN      ] E8SimpleTestCase.E8SimpleTest
truth table has 2 rows
constructing family graph ...
graph has 3 vertices!
>> incoming target si: 0x20634b0
E8Roadmap: constructing 1 batches ...
E8Roadmap: initializing 98 edges ...
E8Roadmap: constructor finished.
Segmentation fault (core dumped)
```

```
$ rosrun ompl_multiset test_e8fromfile 
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from E8FromFileTestCase
[ RUN      ] E8FromFileTestCase.E8FromFileTest
truth table has 2 rows
constructing family graph ...
graph has 3 vertices!
>> incoming target si: 0x19524b0
E8Roadmap: constructing 1 batches ...
E8Roadmap: initializing 98 edges ...
E8Roadmap: constructor finished.
Segmentation fault (core dumped)
```

I think this is an actual bug. :smile: 

**Edit:** Here's the stack trace from `e8simple`:

```
#0  0x000000000040ed57 in ompl_multiset::FamilyEffortModel::x_hat(unsigned long, ompl::base::State const*) ()
#1  0x00007ffff72ed053 in ompl_multiset::E8Roadmap::calculate_w_lazy(boost::detail::edge_desc_impl<boost::undirected_tag, unsigned long> const&) ()
   from /home/mkoval/storage/herb-ws/devel/lib/libompl_multiset.so
#2  0x00007ffff72ef491 in ompl_multiset::E8Roadmap::setProblemDefinition(boost::shared_ptr<ompl::base::ProblemDefinition> const&) () from /home/mkoval/storage/herb-ws/devel/lib/libompl_multiset.so
#3  0x000000000040d800 in E8SimpleTestCase_E8SimpleTest_Test::TestBody() ()
#4  0x00007ffff7541373 in HandleSehExceptionsInMethodIfSupported<testing::Test, void> (location=0x7ffff75440b1 "the test body", method=<optimized out>, object=<optimized out>)
    at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2090
#5  testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void> (object=object@entry=0x622580, method=<optimized out>, location=location@entry=0x7ffff75440b1 "the test body")
    at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2126
#6  0x00007ffff7536c87 in testing::Test::Run (this=this@entry=0x622580) at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2162
#7  0x00007ffff7536d2e in testing::TestInfo::Run (this=0x6220d0) at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2338
#8  0x00007ffff7536e35 in testing::TestCase::Run (this=0x621f60) at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2445
#9  0x00007ffff7539b88 in testing::internal::UnitTestImpl::RunAllTests (this=0x622290) at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:4237
#10 0x00007ffff7539e27 in HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=<optimized out>, method=<optimized out>, object=<optimized out>)
    at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2090
#11 HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=0x7ffff7546570 "auxiliary test code (environments or event listeners)", 
    method=(bool (testing::internal::UnitTestImpl::*)(testing::internal::UnitTestImpl * const)) 0x7ffff7539950 <testing::internal::UnitTestImpl::RunAllTests()>, object=0x622290)
    at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:2126
#12 testing::UnitTest::Run (this=<optimized out>) at /home/mkoval/storage/herb-ws/build/or_trajopt/libtrajopt-prefix/src/libtrajopt/ext/gtest/src/gtest.cc:3874
#13 0x000000000040cbef in main ()
```

**Comment by cdellin** [comment]
Thanks for the report! Fixed in 76ef1d56a57c5aeec2336c55d8523baddb0736f7.
